doxia-modules/doxia-module-xdoc/src/main/java/org/apache/maven/doxia/module/xdoc/XdocSink.java

```diff
@@ -382,119 +377,6 @@ else if ( depth == SECTION_LEVEL_5 )
     //
     // -----------------------------------------------------------------------
 
-    /**
-     * {@inheritDoc}
-     *
-     * @see XdocMarkup#SOURCE_TAG
-     * @see javax.swing.text.html.HTML.Tag#PRE
-     * @param attributes a {@link org.apache.maven.doxia.sink.SinkEventAttributes} object.
-     */
-    public void verbatim( SinkEventAttributes attributes )
-    {
```

Comment:
I don't have a better idea how to make it proper, but it is definitively broken with XHML5.

Reply:
I think this statement is nonsense and this method needs to be retained:
* Xdoc uses element of HTML5, but is not HTML5
* When XdocParser writes to a sink that sink will never see `<source>` anymore.

We have here a mix of custom elements and HTML5.